sktime/utils/validation/forecasting.py

```diff
@@ -293,6 +295,9 @@ def check_fh(fh, enforce_relative=False):
     if enforce_relative and not fh.is_relative:
         raise ValueError("`fh` must be relative, but found absolute `fh`")
 
+    if fh.freq is None:
```

Comment:
why do we need this? Seems superfluous

Reply:
A user or some internal function may pass `ForecastingHorizon` object to `check_fh` which was not initialized with `freq` argument (default = `None`). If so, we update this attribute here.

Reply:
makes sense